### ISSUE TITLE
Docs: learn -> improving-accessibility, missing something about accessiblity 

### ISSUE THREAD

**Issue by realtebo** [vercel/next.js#68884]
### What is the update you wish to see?

I expect no error following tutorial.

I run

     pnpm lint

and instead of 

    ✔ No ESLint warnings or errors

I see 

```
% pnpm lint

> @ lint /Users/mirkotebaldi/Projects/nextjs/nextjs-dashboard
> next lint

Failed to load config "next/typescript" to extend from.
Referenced from: /Users/mirkotebaldi/Projects/nextjs/nextjs-dashboard/.eslintrc.json
 ELIFECYCLE  Command failed with exit code 1.
```

### Is there any context that might help us understand?

I'm using 

    "next": "15.0.0-canary.115",

The `.eslintrc.json` file

```
{
  "extends": [
    "next/core-web-vitals",
    "next/typescript"
  ]
}
```

### Does the docs page already exist? Please link to it.

https://nextjs.org/learn/dashboard-app/improving-accessibility

**Comment by samcx** [comment]
@realtebo Thank you for submitting an issue!

<picture data-single-emoji=":nextjs:" title=":nextjs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/nextjs/2a85d633ae594556.png" alt=":nextjs:" align="absmiddle"></picture> Learn issues should be opened in the other repository instead. 🙇🏼 

![CleanShot 2024-08-15 at 14 21 58@2x](https://github.com/user-attachments/assets/93363a0d-f0a2-405a-94d8-18ddefd7e877)

**Comment by maratkhanbekov** [comment]
it seems that this is due to incompatibility of lint and the typescript version in the project.
Solution 1. You can comment out the line in .eslintrc.json:
```
{
  "extends": [
    "next/core-web-vitals"
    // "next/typescript"
  ]
}
```
Solution 2. Reinstall lint or typescript to a another compatible version

**Comment by psylone** [comment]
> it seems that this is due to incompatibility of lint and the typescript version in the project. Solution 1. You can comment out the line in .eslintrc.json:
> 
> ```
> {
>   "extends": [
>     "next/core-web-vitals"
>     // "next/typescript"
>   ]
> }
> ```
> 
> Solution 2. Reinstall lint or typescript to a another compatible version

I realized that the [`next/typescript`](https://www.npmjs.com/package/eslint-config-next?activeTab=code) linter rules haven't been published yet.
